scripts/functions/requirements/pclinuxos

```diff
@@ -45,20 +57,21 @@ or there was problem checking if libyaml-devel is available / installed."
 # not sure if this one is needed for pclinuxos, copied from centos
 requirements_pclinuxos_update_system_check_epel()
```

Comment:
I actually think this function is not implemented in the right way. Issue reports `PCLinuxOS-2016` and we do not have `epel2016` in `config/db`. So we should figure out another way of checking if system needs to be uodated (or remove the `[[ -z "$epel_rpm" || -z "$epel_key" ]]` part)